From: Alexey Dobriyan <adobriyan@openvz.org>
Date: Mon, 3 Mar 2008 10:46:34 +0000 (+0300)
Subject: CPT: select tun, venet, veth modules
X-Git-Url: http://git.openvz.org/?p=linux-2.6.24-openvz;a=commitdiff_plain;h=b1a7063c226542f1f62972173056fa0971f9d9e8;hp=da23c1831f33ce142b38534f8e54f985f651707c

CPT: select tun, venet, veth modules

CONFIG_VZ_CHEKPOINT=y, CONFIG_TUN=m result in linking breakage
because CPT can't find tun_chr_open() et al.

So, if you build CPT as standalone, tun/tap will be also standalone.
---

diff --git a/kernel/Kconfig.openvz b/kernel/Kconfig.openvz
index 7371679..5cdcdd3 100644
--- a/kernel/Kconfig.openvz
+++ b/kernel/Kconfig.openvz
@@ -71,6 +71,9 @@ config VZ_CHECKPOINT
  	tristate "Checkpointing & restoring Virtual Environments"
  	depends on VE_CALLS && INET
 	select PM_SLEEP
+	select TUN
+	select VE_ETHDEV
+	select VE_NETDEV
  	default m
  	help
  	  This option adds two modules, "cpt" and "rst", which allow
